Makefile

```diff
@@ -651,13 +653,27 @@ pkg/pb/%.pb.go: api/%.proto third_party/ build/toolchain/bin/protoc$(EXE_EXTENSI
 		--go_out=plugins=grpc:$(REPOSITORY_ROOT)/build/prototmp
 	mv $(REPOSITORY_ROOT)/build/prototmp/open-match.dev/open-match/$@ $@
 
+pkg/pb/%.cs: third_party/ build/toolchain/bin/protoc$(EXE_EXTENSION)
+	mkdir -p $(REPOSITORY_ROOT)/build/prototmp/open-match.dev/open-match/pkg/pb $(REPOSITORY_ROOT)/pkg/pb
+	$(PROTOC) api/$(shell echo $(*F)| tr A-Z a-z).proto \
+		-I $(REPOSITORY_ROOT) -I $(PROTOC_INCLUDES) \
+		--csharp_out=$(REPOSITORY_ROOT)/build/prototmp/open-match.dev/open-match/pkg/pb
+	mv $(REPOSITORY_ROOT)/build/prototmp/open-match.dev/open-match/$@ $@
+
 internal/ipb/%.pb.go: internal/api/%.proto third_party/ build/toolchain/bin/protoc$(EXE_EXTENSION) build/toolchain/bin/protoc-gen-go$(EXE_EXTENSION) build/toolchain/bin/protoc-gen-grpc-gateway$(EXE_EXTENSION)
 	mkdir -p $(REPOSITORY_ROOT)/build/prototmp $(REPOSITORY_ROOT)/internal/ipb
 	$(PROTOC) $< \
 		-I $(REPOSITORY_ROOT) -I $(PROTOC_INCLUDES) \
 		--go_out=plugins=grpc:$(REPOSITORY_ROOT)/build/prototmp
 	mv $(REPOSITORY_ROOT)/build/prototmp/open-match.dev/open-match/$@ $@
 
+internal/ipb/%.cs: third_party/ build/toolchain/bin/protoc$(EXE_EXTENSION)
```

Comment:
We shouldn't be building internal c# protos.  All of the core OM (the only thing that should be talking to internal containers) are written in Go.